ABPConsumingRestApiByUsingStaticProxy/README.md

```diff
@@ -0,0 +1,244 @@
+        {
+            var bookDtos = new List<BookDto>()
+            {
+                new BookDto(){ Name = "Anna Karenina", AuthorName ="Tolstoy", Price = 50},
+                new BookDto(){ Name = "Crime and Punishment", AuthorName ="Dostoevsky", Price = 60},
+                new BookDto(){ Name = "Mother", AuthorName ="Gorki", Price = 70}
+            };
+            return Task.FromResult(new PagedResultDto<BookDto>(
+               bookDtos.Count,
+               bookDtos
+           ));
+        }
+    }
+}
+```
+It simply returns a list of books. You probably want to get the books from a database, but it doesn't matter for this article. To do it you can visit [here](https://docs.abp.io/en/abp/latest/Tutorials/Part-1?UI=MVC&DB=EF)
+
+### Consume the app service from the console application
+Change `ClientDemoService` as shown the following in the `Acme.BookStore.HttpApi.Client.ConsoleTestApp` project under the test folder.
+
+```csharp
+using Acme.BookStore.Books;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Volo.Abp.Application.Dtos;
+using Volo.Abp.DependencyInjection;
+
+namespace Acme.BookStore.HttpApi.Client.ConsoleTestApp;
+
+public class ClientDemoService : ITransientDependency
+{
+    private readonly IBookAppService _bookAppService;
+
+    public ClientDemoService(IBookAppService bookAppService )
+    {
+        _bookAppService = bookAppService;
+    }
+
+    public async Task RunAsync()
+    {
+        var listOfBooks = await _bookAppService.GetListAsync(new PagedAndSortedResultRequestDto());
+        Console.WriteLine($"Books: {string.Join(", ", listOfBooks.Items.Select(p => p.Name).ToList())}");
+    }
+}
+```
+
+The output should be 
+> Books: Anna Karenina, Crime and Punishment, Mother
+
+### Convert application to use static client proxies
+Before showing you how to use static client proxies instead of dynamic client proxies, I ask you to talk differences between both approaches. Their similarities, advantages and disadvantages to each other.
+
+##### Benefits
+* Maps C# method calls to remote server HTTP calls by considering the HTTP method, route, query string parameters, request payload and other details.
+* Authenticates the HTTP Client by adding access token to the HTTP header.
+* Serializes to and deserialize from JSON.
+* Handles HTTP API versioning.
+* Add correlation id, current tenant id and the current culture to the request.
+* Properly handles the error messages sent by the server and throws proper exceptions.
+
+##### Differences
+Static generic proxies provide better performance because it doesn't need to run on runtime, but you should **re-generate** once changing API endpoint definition. Dynamic generic proxies don't need **re-generate** again because it works on the runtime but it happens more a bit time. 
+
+Now focus on how to do it,
+Firstly add Volo.Abp.Http.Client NuGet package to your client project:
+````shell
+Install-Package Volo.Abp.Http.Client
```

Comment:
Is this required? Doesn't it be included indirectly by BookStore.HttpApi.Client?